src/ui/components/touchableLink/TouchableLink.tsx

```diff
@@ -32,8 +34,8 @@ export function TouchableLink({
   ) as ElementType
   const TouchableLinkComponent = Tag ? Tag : TouchableComponent
   const linkRef = createRef<HTMLAnchorElement>()
-  const [isFocus, setIsFocus] = useState(false)
-  const [isHover, setIsHover] = useState(false)
+  const { onFocus: onFocusDefault, onBlur: onBlurDefault, isFocus } = useHandleFocus()
+  const { onMouseEnter, onMouseLeave, isHover } = useHandleHover()
```

Comment:
Les changements de ce commit ((TouchableLink) use Hover and focus from new hooks) me paraissent hors scope. Qu'est-ce qui les a motivé ?

Reply:
En fait c'était des changements qu'Hugo avait fait sur le touchableLink dans une autre PR qui a été merge cette semaine. Comme moi j'ai créé un touchableLink2 en reprenant le code de TouchableLink il y a un mois je n'avais pas ces changements qui étaient pourtant importants (je les ai juste copié collé de sa PR)

Reply:
En fait ils aurait du être dans le commit de séparation des API et de création du touchableLink2